hypervisor/network/network.go

```diff
@@ -9,19 +9,20 @@ import (
 )
 
 type Settings struct {
-	Mac         string
-	IPAddress   string
-	IPPrefixLen int
-	Gateway     string
-	Bridge      string
-	Device      string
-	File        *os.File
-	Automatic   bool
+	Mac       string
+	IPAddress []string
+	Mtu       uint64
+	Gateway   string
+	Bridge    string
+	Device    string
+	File      *os.File
+	Automatic bool
 }
 
 const (
 	DefaultBridgeIface = "hyper0"
 	DefaultBridgeIP    = "192.168.123.0/24"
+	DefaultMtu         = 1450
```

Comment:
I remember that vxlan interface has a mtu of 1450, once I met a problem that container network can't visit outside via vxlan, after setting its mtu to 1450, everything is good again.

But I'll keep mtu empty to let os handle it as your other comments suggest.